src/server/pipeline/flow_script.py

```diff
@@ -25,7 +25,7 @@ def start_flow():
 
 
         if file_path_list:
-            with engine.connect() as connection:
+            with engine.begin() as connection:
                 Base.metadata.create_all(connection)
 
                 # Get previous version of pdp_contacts table, which is used later to classify new records
```

Comment:
How does the pdp_contacts table get created on a new database?

Reply:
I'll create an alembic script for the initial creation of the table

Reply:
Do you think this makes sense as part of the initial schema script or are we considering the database live and I should create a new script for this. The table creation does happen in `models.py` but I think we should keep all database maintenance in one place and since alembic is a more complete solution any remaining functionality should be move into alembic scripts.

Reply:
>  I think we should keep all database maintenance in one place and since alembic is a more complete solution any remaining functionality should be move into alembic scripts.

I agree. Explicit is much better than implicit/magical.

Reply:
Assigned #395 to you as this seems the right place to do it.